### PR TITLE
all: clean up goerli flag and config

### PR DESCRIPTION
**Pull request by stevemilk**

### CODE REVIEW

**Comment by lightclient** [comment]
Can you update the readme to direct users to holesky instead of goerli?

**Comment by stevemilk** [comment]
> Can you update the readme to direct users to holesky instead of goerli?

@lightclient readme docs updated.

**Review by rjl493456442 — APPROVED** [review]

File: internal/era/era_test.go [hunk 1]
```diff
@@ -132,7 +132,6 @@ func TestEraFilename(t *testing.T) {
 		expected string
 	}{
 		{"mainnet", 1, common.Hash{1}, "mainnet-00001-01000000.era1"},
-		{"goerli", 99999, common.HexToHash("0xdeadbeef00000000000000000000000000000000000000000000000000000000"), "goerli-99999-deadbeef.era1"},
```
Comment (MariusVanDerWijden) [inline internal/era/era_test.go:135]: We should have some more tests than only mainnet here

**Comment by lightclient** [comment]
going to close to see if reopening will restart gitguardian

**Review by lightclient — APPROVED** [review]

**Pull request merged**